components/link/stories/link.test.js

```diff
@@ -48,6 +48,15 @@ export const LinkGroup = Variants({
 			staticColor: "white",
 			isQuiet: true,
 		},
+		{
```

Comment:
I like that you added these at the bottom because they're new but for S2, I'm wondering what you think of listing the inline variant right under the default because it's a more central style setting than say, the static color tests.